packages/x-date-pickers/src/locales/daDK.ts

```diff
@@ -0,0 +1,77 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+
+const daDKPickers: Partial<PickersLocaleText<any>> = {
+  // Calendar navigation
+  previousMonth: 'Forrige måned',
+  nextMonth: 'Næste måned',
+
+  // View navigation
+  openPreviousView: 'åben forrige visning',
+  openNextView: 'åben næste visning',
+  calendarViewSwitchingButtonAriaLabel: (view) =>
+    view === 'year'
+      ? 'årsvisning er åben, skift til kalendervisning'
+      : 'kalendervisning er åben, skift til årsvisning',
+
+  // DateRange placeholders
+  start: 'Start',
+  end: 'Slut',
+
+  // Action bar
+  cancelButtonLabel: 'Annuller',
+  clearButtonLabel: 'Ryd',
+  okButtonLabel: 'OK',
+  todayButtonLabel: 'I dag',
+
+  // Toolbar titles
+  datePickerToolbarTitle: 'Vælg dato',
+  dateTimePickerToolbarTitle: 'Vælg dato & tidspunkt',
+  timePickerToolbarTitle: 'Vælg tidspunkt',
+  dateRangePickerToolbarTitle: 'Vælg datointerval',
+
+  // Clock labels
+  clockLabelText: (view, time, adapter) =>
+    `Vælg ${view}. ${
```

Comment:
Please refer to `deDE`, `frFR`, or any other locale, where the view is translated for an example of how to handle this case.
Currently, the `view` would always be the English word here. 🤔